### PR TITLE
Reagent atom from theme regression #19729

### LINKED ISSUE #19720: status bar is black in a dark theme
Devices: iOS 13 (17.3.1) Samsung A34 (14.0)

Steps:

- Login to app
- Go to Profile-> Appearance
- Change theme to dark
- Return to Messages or Community screens

Actual result:

image

### LINKED ISSUE #19729: Reagent atom from theme regression
Could it be regression of https://github.com/status-im/status-mobile/pull/19706 ?  cc @flexsurfer 
Let me know if some of the issues are not regression of https://github.com/status-im/status-mobile/pull/19706 , I will log separately.

### ISSUE 1 Wrong font colour on recovery phrase screen

**Steps:** 

1. Open recovery phrase screen
2. Start typing seed phrase
3. Pay attention at font colour

**Expected result:** font colour should be white

**Actual result:** font colour is black

photo_2024-04-19 14 17 44

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

### ISSUE 2 Wrong font colour of amount value

**Steps:** 

1. Open asset amount screen
2. Start typing some decimal value, i.e 0.001 
3. Pay attention at value font colour

**Actual result:** colour is white

https://github.com/status-im/status-mobile/assets/97245802/cef5a215-b2eb-4331-a016-7c716b37873d

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #19729

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]
Thank you for the great work.

File: src/status_im/navigation/effects.cljs [hunk 1]
```diff
@@ -214,6 +215,7 @@
 
 (rf/reg-fx :show-toasts
  (fn [[view-id theme]]
+   (println "show toask " view-id theme)
```
Comment (Parveshdhull) [inline src/status_im/navigation/effects.cljs]: leftover?

**Review by Parveshdhull — APPROVED** [review]

**Comment by pavloburykh** [comment]
@flexsurfer thanks for the PR. Please take a look at the issue. 

### ISSUE 1 "nth not supported on this type function.." after onboarding

**Steps:** 

1. Pass onboarding
2. Pay attention at the error after onboarding

**Actual result:**

photo_2024-04-22 13 16 53

https://github.com/status-im/status-mobile/assets/97245802/9be23e1a-273e-4c76-8189-e58d3141c4d5

**Comment by qoqobolo** [comment]
@flexsurfer friendly reminder: do not forget to smoke-check your changes please (at least install the app and create a profile) 🙂🙏
Thank you!

**Review by J-Son89 — APPROVED** [review]

**Comment by flexsurfer** [comment]
sorry @pavloburykh , thank you, should be fixed now, but lets wait e2e first

**Comment by yevh-berdnyk** [comment]
Hi, e2e failures are not PR related 👍

**Comment by pavloburykh** [comment]
@flexsurfer thanks for the PR. Most of the issues are fixed. 

Although, I am still reproducing the following

### ISSUE 1 https://github.com/status-im/status-mobile/issues/19720 status bar is black in a dark theme

photo_2024-04-23 17 52 46

### ISSUE 2 Black circle around community avatar

Originally reported [here](https://github.com/status-im/status-mobile/issues/19729#issuecomment-2066705355)

photo_2024-04-23 17 55 08

**Comment by flexsurfer** [comment]
thanks @pavloburykh fixed 12 and removed status  bar from fixes

**Comment by pavloburykh** [comment]
@flexsurfer thanks for the fixes. PR is ready for merge.

**Pull request merged**